mods/ra/maps/allies-06a/allies06a.lua

```diff
@@ -115,6 +115,19 @@ CaptureRadarDome = function()
 
 	Trigger.OnCapture(Radar, function()
 		player.MarkCompletedObjective(CaptureRadarDomeObj)
+		Beacon.New(player, TechLab1.CenterPosition)
+		Beacon.New(player, TechLab2.CenterPosition)
+		Beacon.New(player, TechLab3.CenterPosition)
+		Media.DisplayMessage("We discovered the coordinations of the Soviet tech centers.")
```

Comment:
I'm not sure that this message adds anything to the mission.  Does anyone else have an opinion here?

Reply:
Not a strong one, but imo the message doesn't hurt, either.

I'd keep it a bit more concise in the tone of the original though, something like `Tech center coodinates discovered`.

Reply:
I changed it to "Coordinates of the Soviet tech centers discovered." because it sounds a little bit better than "Soviet tech centers coordinates discovered." to me. There are multiple tech centers.

But I have to say, English is not my forte.